modules/vaos/app/services/vaos/jwt_wrapper.rb

```diff
@@ -38,25 +38,13 @@ def payload
         gender: gender,
         dob: parsed_date,
         dateOfBirth: parsed_date,
-        edipid: edipi,
-        ssn: ssn
+        edipid: user.edipi,
+        ssn: user.ssn
       }
     end
 
-    def icn
-      user.icn
-    end
-
-    def first_name
-      user.mpi&.profile&.given_names&.first
```

Comment:
@saneshark Did you see these changes?  I assume you made an active choice to get the data from MPI over the user object.

Reply:
We made an active choice to derive from MPI because VAMF uses MPI. It's necessary that we provide the MPI values here vs whatever might be in user since not all paths to LOA3 are necessarily derived from IAM / SSOE.

Reply:
The JWT that we send to VAMF to authenticate against their APIs will only work if it is consistent with what they can lookup via ICN on their side. 

For example on staging there is a user Judy Morrison, when you sign in as this user first_name is "Judy", but MPI shows "JUDY." These types of differences would cause issues with our ability to authenticate against the VAMF services we are integrated with.

Reply:
I would have liked to comment on these changes prior to them getting merged since it affects VAOS services. Can we make sure to include VAOS backend team when making changes to modules/vaos/* in the future?

Reply:
Absolutely, I'll make sure to include you and @annaswims when I submit the refactored pull request and anything else that touches the VAOS module.